web/src/components/pages/contribution/listen/listen.tsx

```diff
@@ -128,10 +131,38 @@ class ListenPage extends React.Component<Props, State> {
 
   private vote = (isValid: boolean) => {
     const { clips } = this.state;
+    const { achievement, addNotification } = this.props;
     const clipIndex = this.getClipIndex();
 
     this.stop();
     this.props.vote(isValid, this.state.clips[this.getClipIndex()].id);
+    if (achievement) {
+      addNotification(
+        <div className="achievement">
+          <img
+            src={require('../../dashboard/challenge/images/star.svg')}
+            alt=""
+          />
+          <p className="score">+ 50 points</p>
+          <p>You're on your way! Congrats on your first contribution. </p>
+        </div>
+      );
```

Comment:
When a user has finished his first contribution, the back-end will send `achievement: true` on the request and trigger a toast immediately

Reply:
As discussed, it is not `achievement: true `any more, it is `firstContribute: true`